app/components/provider_interface/activity_log_event_component.rb

```diff
@@ -95,6 +118,11 @@ def link
             url: routes.provider_interface_application_choice_offer_path(application_choice),
             text: 'View offer',
           }
+        elsif event.audit.auditable.is_a?(Interview)
+          {
+            url: routes.provider_interface_application_choice_interviews_path(event.audit.associated, anchor: "interview-#{event.audit.auditable.id}"),
```

Comment:
@despo I've added an anchor using the interview ID for now, though it would be possible to use the interview date and time if this isn't suitable.